src/Domain/Bootstrap.php

```diff
@@ -139,6 +141,7 @@ function() {
 			$this->container->get( AssetsController::class );
 			$this->container->get( Installer::class )->init();
 			$this->container->get( GoogleAnalytics::class )->init();
+			$this->container->get( CheckoutFields::class )->init();
```

Comment:
While we register the class early on, we only init it here to pass data to Checkout block via wcSettings.